archetypes/jbehave-simple-archetype/src/main/resources/archetype-resources/src/main/java/MyStories.java

```diff
@@ -61,9 +61,11 @@ public Configuration configuration() {
         ParameterControls parameterControls = new ParameterControls();
         // Start from default ParameterConverters instance
         ParameterConverters parameterConverters = new ParameterConverters(resourceLoader, tableTransformers);
+        // A simple table parser
+        TableParsers tableParser =  new TableParsers();
```

Comment:
There is no no-args constructor for the TableParsers https://github.com/jbehave/jbehave-core/blob/master/jbehave-core/src/main/java/org/jbehave/core/model/TableParsers.java#L29